Automattic/wc-calypso-bridge
# Free trial: Hide partial tasklist and tasks

Partially fixes https://github.com/Automattic/wc-calypso-bridge/issues/945

## Changes in this PR

- Hide "Add a domain" (AddDomain) task
- Hide "Get more sales" (Marketing) task
- Hide "Things to do next" tasklist


## Test instructions

1. Make sure to copy over the latest changes in [woocmmerce-calypso-bridge-helper.php](https://gist.github.com/moon0326/cac46c70a2cee81b61faef517fef7178) and the plugin is activated.
2. Change [this line](https://gist.github.com/moon0326/cac46c70a2cee81b61faef517fef7178#file-a-woocommerce-calypso-bridge-helper-php-L33) to false in your local env. This forces your env to have free trial.
3. Go to WooCommerce > Home
4. Confirm domain, marketing task, and "Things to do next" tasklist are hidden

![Screenshot 2023-02-16 at 10 17 09](https://user-images.githubusercontent.com/4344253/219250760-d33abab1-c4c8-4c5c-8257-5ec0447f0a9d.png)
